packages/volto/src/components/theme/Register/Register.stories.jsx

```diff
@@ -20,6 +20,32 @@ function StoryComponent(args) {
           locale: 'en',
           messages: {},
         },
+        userschema: {
+          loaded: true,
+          loading: false,
+          userschema: {
+            fieldsets: [
+              {
+                id: 'default',
+                title: 'default',
+                fields: ['fullname', 'email'],
+              },
+            ],
+            properties: {
+              fullname: {
+                type: 'string',
+                title: 'Fullname',
```

Comment:
For text displayed to the user, "full name" is two words. Also make similar replacements in the test files.

```suggestion
                title: 'Full name',
```

Reply:
@erral I unresolved this suggestion, as there are other instances to replace.

> Also make similar replacements in the test files.

Reply:
Sorry I overlooked the suggestion.